semgrep_output_v1.atd

```diff
@@ -89,14 +89,21 @@ type location
 }
 
 (*****************************************************************************)
-(* Rule id *)
```

Comment:
I would have kept Rule id for the section comment
We're not defining rules here.